### DISCUSSION TITLE
API Documentation

### DISCUSSION THREAD

**Discussion by okpatil4u** [kagisearch/kagi-docs#24] [Kagi Search API Feature Requests & Ideas]
Hello there,

I just subscribed to individual plan and blown away by lenses. I am interested in adding my teammates into this search ecosystem.

Where can I find the search API documentation ? Are lenses supported there ?

**Comment by okpatil4u** [comment]
My apologies, seems like the API documentation is here.

https://help.kagi.com/kagi/search-api/endpoints/execute-search.html

Is there any way I could add lense to the search query ?

**Reply by vprelovac** [reply]
I believe we do not support lenses in the programatic API yet, but this should be easilly achievable. Can you please open a feature suggestion at https://kagifeedback.org ?

**Comment by okpatil4u** [comment]
Thanks for your reply. I tried to register on the portal you suggested,
it’s giving me an error “turnstile field required”. I will try again.

Just a followup question, what kind of timeline do you think it would take
? I would be happy to pay per search query once this feature is online.

**Reply by vprelovac** [reply]
turnstile is a bot-protection mechanism. Perhaps try with a different browser?

We could have this as early as January.

**Comment by okpatil4u** [comment]
Thanks, I just requested the feature in the portal.
